app/src/main/res/layout/fragment_more_track.xml

```diff
@@ -0,0 +1,96 @@
+<layout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    xmlns:app="http://schemas.android.com/apk/res-auto">
+    <data>
+        <variable
+            name="viewModel"
+            type="com.framgia.soundclound.screen.moretrack.MoreTrackViewModel"/>
+
+    </data>
+
+    <android.support.constraint.ConstraintLayout
+        android:layout_width="match_parent"
+        android:layout_height="@dimen/dp_200">
+
+        <ImageView
+            android:contentDescription="@string/title_track"
+            android:id="@+id/image_view_more"
+            android:layout_marginTop="@dimen/dp_16"
+            app:layout_constraintLeft_toLeftOf="parent"
+            app:layout_constraintTop_toTopOf="parent"
+            app:layout_constraintRight_toLeftOf="@id/text_view_title_track"
+            android:layout_width="@dimen/dp_60"
+            android:layout_height="@dimen/dp_60" />
+
+        <TextView
+            android:lines="1"
+            android:textSize="@dimen/sp_14"
+            android:layout_marginStart="@dimen/dp_24"
+            android:layout_marginTop="@dimen/dp_20"
+            android:text="@{viewModel.title}"
+            android:id="@+id/text_view_title_track"
+            app:layout_constraintLeft_toRightOf="@id/image_view_more"
+            app:layout_constraintTop_toTopOf="parent"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content" />
+
+        <TextView
+            android:lines="1"
+            android:text="@{viewModel.artist}"
+            android:id="@+id/text_view_artist"
+            android:textSize="@dimen/sp_12"
+            android:layout_marginTop="@dimen/dp_12"
+            app:layout_constraintTop_toBottomOf="@id/text_view_title_track"
+            android:layout_marginStart="@dimen/dp_24"
+            app:layout_constraintLeft_toRightOf="@id/image_view_more"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content" />
+        
+        <ImageView
+            android:contentDescription="@string/title_like"
+            android:src="@drawable/ic_favorite_border_black_24dp"
+            android:onClick="@{viewModel::onClickLike}"
+            android:id="@+id/image_view_like"
+            app:layout_constraintLeft_toLeftOf="parent"
+            android:layout_marginLeft="@dimen/dp_28"
+            android:layout_marginTop="@dimen/dp_14"
+            app:layout_constraintTop_toBottomOf="@id/image_view_more"
+            android:layout_width="@dimen/dp_30"
+            android:layout_height="@dimen/dp_30" />
+
+        <TextView
+            android:text="@string/title_add_favorite"
+            android:id="@+id/text_view_like"
+            android:layout_marginLeft="@dimen/dp_18"
```

Comment:
WARNING: [RtlHardcoded] Consider replacing `android:layout_marginLeft` with `android:layout_marginStart="@dimen/dp_18"` to better support right-to-left layouts
 Using `Gravity#LEFT` and `Gravity#RIGHT` can lead to problems when a layout is rendered in locales where text flows from right to left. Use `Gravity#START` and `Gravity#END` instead. Similarly, in XML `gravity` and `layout_gravity` attributes, use `start` rather than `left`.

For XML attributes such as paddingLeft and `layout_marginLeft`, use `paddingStart` and `layout_marginStart`. **NOTE**: If your `minSdkVersion` is less than 17, you should add **both** the older left/right attributes **as well as** the new start/right attributes. On older platforms, where RTL is not supported and the start/right attributes are unknown and therefore ignored, you need the older left/right attributes. There is a separate lint check which catches that type of error.

(Note: For `Gravity#LEFT` and `Gravity#START`, you can use these constants even when targeting older platforms, because the `start` bitmask is a superset of the `left` bitmask. Therefore, you can use `gravity="start"` rather than `gravity="left|start"`.)